docs/README.adoc

```diff
@@ -0,0 +1,31 @@
+= How to work on documentation
+
+The documentation is all in link:http://www.methods.co.nz/asciidoc/[asciidoc].
+
+== Build
+
+The documentation is built with
+
+  ./gradlew :docs:asciidoctor
+
+You will then find the result in `docs/build/asciidoc/user`.
+
+== Live reload
+
+To get live reload while editing the documentation, you need two shells.
+
+In the first one, do a continuous buid of the documentation.
+
+  ./gradlew -t :docs:asciidoctor
+
+In the second launch the live reload.
```

Comment:
I'm not convinced that the live reload stuff belongs in the buildscript itself.

Reply:
Of course it does. It's nicely configured and not doing anything unless the task is launched.